scripts/generate_data_in_parallel.sh

```diff
@@ -23,10 +27,22 @@ if [ -z "$CONFIG_PATH" ] || [ -z "$N_FILES_PER_PROCESS" ]; then
     exit 1
 fi
 
+pids=()
 for ((i=0; i<$NUM_PROCESSES; i++));
 do
     files_start_idx=$((i * N_FILES_PER_PROCESS))
-    bliss -cp $CONFIG_PATH 'mode=generate' "simulator.num_workers=$N_WORKERS_PER_PROCESS" "generate.files_start_idx=$files_start_idx" &
+    bliss -cp $CONFIG_PATH 'mode=generate' "simulator.num_workers=$N_WORKERS_PER_PROCESS" "generate.files_start_idx=$files_start_idx" $CONFIG_OVERRIDES &
     pids+=($!) # store PID in array
 done
-wait
+
+# wait for all bliss processes to finish
+for pid in "${pids[@]}"; do
+    wait $pid
+done
+
+cleanup() {
+    echo "Killing bliss data generation processes..."
+    for pid in "${pids[@]}"; do
+        kill $pid
+    done
+}
```

Comment:
I actually added in the cleanup last week when you were out - can you remove that from this PR?
https://github.com/prob-ml/bliss/blob/95ad90f651765118255762781200bee420e03969/scripts/generate_data_in_parallel.sh#L9-L18

Reply:
Fixed.